graphql/resolvers.go

```diff
@@ -800,6 +934,33 @@ func (r *queryResolver) MyHosts(ctx context.Context) ([]*restModel.APIHost, erro
 	return apiHosts, nil
 }
 
+func (r *queryResolver) ProjectSettings(ctx context.Context, identifier string) (*restModel.APIProjectSettings, error) {
+	// first, check project collection
+	res := &restModel.APIProjectSettings{}
+	projectRef, err := model.FindOneProjectRef(identifier)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error looking in project collection: %s", err.Error()))
+	}
```

Comment:
nit: can we maybe add a 
// then, check the repo collection 
here to close the loop?